pkg/apis/eksctl.io/v1alpha5/defaults.go

```diff
@@ -325,3 +325,11 @@ func (c *ClusterConfig) SetDefaultFargateProfile() {
 		},
 	}
 }
+
+// SetDefaultKarpenterCreateServiceAccount disables the create service account option for Karpenter if it's not
+// defined in the configuration file explicitly.
+func (c *ClusterConfig) SetDefaultKarpenterCreateServiceAccount() {
```

Comment:
I find the naming a bit verbose. WDYT about calling it just `SetKarpenterDefaults`?